R/theme_gtsummary_roche.R

```diff
@@ -17,7 +18,7 @@
 #' tbl_demographics(trial, by = trt, include = c(age, grade))
 #'
 #' reset_gtsummary_theme()
-theme_gtsummary_roche <- function(set_theme = TRUE, font_size = NULL) {
```

Comment:
after we take the compact theme calls from gtsummary and add them here, i don't think we'll need the font_size argument. let's remove it and hardcode the font size.